CHANGELOG.md

```diff
@@ -3,6 +3,79 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [4.0.0-alpha.33](https://github.com/mui-org/material-ui-x/compare/v4.0.0-alpha.32...v4.0.0-alpha.33)
+
+_July 1, 2021_
+
+Big thanks to the 6 contributors who made this release possible. Here are some highlights ✨:
+
+- 💅 Allow to copy the selected rows to the clipboard (#1929) @m4theushw
+- 🚀 `Select all` checkbox click should select only filtered rows after filter (#1879) @ZeeshanTamboli
+- ⚡️ Add single select column type (#1956) @DanailH@
+- 🐞 Bugfixes
+
+### @material-ui/x-grid@v4.0.0-alpha.32 / @material-ui/data-grid@v4.0.0-alpha.32
+
+#### Breaking changes
+
+- [DataGrid] Rename onColumnResizeCommitted to onColumnWidthChange (#1967) @m4theushw
+
+```diff
+  -<DataGrid onColumnResizeCommitted={...} />
+  +<DataGrid onColumnWidthChange={...} />
+```
+
+- [DataGrid] Make GRID_ROWS_CLEAR private (#1925) @oliviertassinari
+
+  The `GRID_ROWS_CLEAR` event was always triggered alongside `GRID_ROWS_SET`. You can listen to the latter event only.
+
+- [DataGrid] Fix events naming (#1862) @m4theushw
+
+  The following `XGrid` events were renamed:
+
+  - `columnHeaderNavigationKeydown` to `columnHeaderNavigationKeyDown`
+  - `columnResizeCommitted` to `columnWidthChange`
+  - `rowsUpdated` to `rowsUpdate`
+  - `rowsCleared` to `rowsClear`
+  - `columnsUpdated` to `columnsChange`
+
+  The following `XGrid` DOM events were removed:
+
+  - `focusout`
+  - `keydown`
+  - `keyup`
+
+
+#### Changes
+
+- [DataGrid] Add fallback for pagination translations (#2006) @m4theushw
+- [DataGrid] Add single select column type (#1956) @DanailH
+- [DataGrid] Allow to copy the selected rows to the clipboard (#1929) @m4theushw
+- [DataGrid] Better fix the scrollToIndexes logic (#1969) @oliviertassinari
+- [DataGrid] Fix deferred rendering race condition (#1807) @dtassone
+- [DataGrid] Fix double-click issue (#1919) @oliviertassinari
+- [DataGrid] Fix number edit cell output (#1959) @oliviertassinari
+- [DataGrid] Fix scrollToIndexes offscreen row (#1949) @oliviertassinari
+- [DataGrid] Ignore drag events when disableColumnReorder is true (#1952) @m4theushw
+- [DataGrid] `Select all` checkbox click should select only filtered rows after filter (#1879) @ZeeshanTamboli
+- [XGrid] Add option to select only visible rows on the current page (#1998) @DanailH
+
+### Docs
+
+- [docs] Align docs with EULA (source of truth) (#1963) @oliviertassinari
+- [docs] Fix changing Dataset not working (#1965) @m4theushw
+- [docs] Fix description of union types (#2003) @m4theushw
+- [CHANGELOG] Fix typo @oliviertassinari
+
+### Core
+
+- [core] Polish filtering internals (#1760) @ZeeshanTamboli
+- [core] Upgrade actions-cool/issues-helper (#1962) @oliviertassinari
+- [core] virtualised => virtualized (#1988) @flaviendelangle
```

Comment:
```suggestion
- [core] Name variables according to enUS instead of enGB (#1988) @flaviendelangle
```

Reply:
@flaviendelangle Note that when you merge a PR, try to keep the title for the commit message. It's not done automatically by GitHub if there a single commit.